Templates/Database/StoredProcedures/StoredProcedures.cst

```diff
@@ -1113,7 +1113,10 @@ public string GetBySuffix(IList<ColumnSchema> columns)
     System.Text.StringBuilder bySuffix = new System.Text.StringBuilder();
     foreach(var column in columns.AsIndexedEnumerable())
 	{
-	    if (!column.IsFirst) bySuffix.Append("And");
+	if (column.Index > 0) { 

```

Comment:
Isn't the same as what it was before? Does the IsFirst column check not work?

Reply:
I think there was some kind of fat finger in my copy.  I just pulled everything again and now I get the same results either way. It looked like it would be the same but i was getting the test results from VistaDBPetshop.  Again, I guess there was a change in the version I was test vs the clean OEM version. Sorry to waste your time. I vote we resolve and do not merge.

Reply:
Ok, thank you. :-)